cluster/kubemark/config-default.sh

```diff
@@ -22,7 +22,7 @@
 GCLOUD=gcloud
 ZONE=${KUBE_GCE_ZONE:-us-central1-b}
 MASTER_SIZE=${MASTER_SIZE:-n1-standard-4}
-NUM_MINIONS=${NUM_MINIONS:-100}
+NUM_MINIONS=${NUM_MINIONS:-4000}
```

Comment:
OK, I think I finally understand how this all works. should probably make sure the oncall knows, though, since this will start spamming them due to being broken.